app/src/main/res/layout/fragment_mood.xml

```diff
@@ -61,62 +61,77 @@
     <androidx.constraintlayout.widget.ConstraintLayout
         android:id="@+id/clNotEnoughDays"
         android:layout_width="match_parent"
-        android:layout_height="wrap_content"
-        android:layout_marginStart="15dp"
-        android:layout_marginTop="20dp"
-        android:layout_marginEnd="15dp"
+        android:layout_height="0dp"
         android:visibility="gone"
-        android:background="@drawable/slight_rounded_corners"
-        android:backgroundTint="@color/peach"
+        android:background="@color/cream"
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toBottomOf="@id/ivAverageMood">
+        app:layout_constraintTop_toBottomOf="@id/ivAverageMood"
+        app:layout_constraintBottom_toBottomOf="parent"
+        >
 
-        <TextView
-            android:id="@+id/tvNotEnoughDays"
+        <androidx.constraintlayout.widget.ConstraintLayout
```

Comment:
why do we have ConstraintLayout inside ConstraintLayout?

Reply:
Part of this view is a card that either displays a graph if they have tracked for long enough or displays a card with a few textViews if they have not tracked for long enough. I like to make the "card" a ConstraintLayout so I can easily collapse it as a whole and set the background color so it appears visually as a single "card", but I would be open to any ideas to change this approach.